src/types/todo.ts

```diff
@@ -0,0 +1,5 @@
+export interface Todo {
+  id: string,
+  title: string,
+  completed: true | false,
```

Comment:
```suggestion
  completed: boolean,
```
boolean type is the same